.serena/memories/codebase_structure.md

```
@@ -0,0 +1,42 @@
+# Codebase Structure
+
+## Main Directory Layout
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)